html/semantics/interfaces.html

```diff
@@ -27,7 +33,9 @@
 }
 
 elements.forEach(function(a) {
-  do_test(a[0], a[1]);
-  do_test(a[0].toUpperCase(), a[1]);
+  do_test(a[0], a[1], true);
+  if (a[0] === a[0].toLowerCase()) {
```

Comment:
Because `createElementNS('http://www.w3.org/1999/xhtml', 'H1')` is *not* an HTML `h1` element, so won't have the right interface.